libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/TunnelInfo.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+/**
+ * Tunnel information.
+ */
+class TunnelInfo private constructor(
```

Comment:
Think we should delete this entire class, in favor of a tunnelName: String property in `TunnelEntranceAlert`

The TunnelEntranceAlert has all the tunnel info

Reply:
For consistency was trying to mimic `CountryBorderCrossingAdminInfo` as don't know if extra tunnel info will be included in the future. Also NN `PassiveManeuverTunnelInfo` is currently a class. cc @danpaz @SiarheiFedartsou

Reply:
BTW these objects are given to the clients, we create them internally and clients only access to their properties e.g. `info?.name` https://github.com/mapbox/mapbox-navigation-android/blob/b055dbe091cce0c310d3f0077cdbc35e0668fdfb/examples/src/main/java/com/mapbox/navigation/examples/core/RouteAlertsActivity.kt#L248

But yeah I see your point. If not future info is going to be included the `Builder` is over-killing.

Reply:
hrm can see the argument PassiveManeuverTunnelInfo

thought we changed the name to RouteAlerts? why isn't it `RouteAlertTunnelInfo`?

Reply:
Not yet https://github.com/mapbox/mapbox-navigation-android/pull/3604 😉

Reply:
`TunnelInfo` it is then 👍 . easier to keep consistent with iOS if we're modeling close to what native is doing

Reply:
Also since objects like RouteAlertTunnelInfo are being generated, maybe it's worth generating them as part of the SDK so we remove an extra conversion

Reply:
I would like to re-challenge this decision. The `TunnelInfo` is basically the same pattern as the `Metadata` we've previously removed. In my opinion, it should be a "tunnelName: String property in `TunnelEntranceAlert`" like @kmadsen mentioned.

`CountryBorderCrossingAdminInfo` is a separate object because it's always a pair of values (`code` and `code_alpha3`) so it felt cleaner to introduce a separate data construct (adding a builder to it might've been an overkill though).

This is a minor issue, not a functional problem so I think it's going to be fine to leave it as is, just a note for when we expose other route alerts in the future.